retrofit/src/main/kotlin/com/smartnsoft/ws/retrofit/bo/LoginBody.kt

```diff
@@ -0,0 +1,11 @@
+package com.smartnsoft.ws.retrofit.bo
+
+/**
+ * @author Anthony Msihid
+ * @since 2019.08.23
+ */
+data class LoginBody
+(
+    val username: String,
```

Comment:
We should have Jackson annotation here. It'll be easier to understand what's going on and avoid possible issues with proguard